src/pages/UserAccount/User.js

```diff
@@ -0,0 +1,152 @@
+import React, { useEffect, useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import './SignUp.scss';
+
+function User({ userData, isSelectLogin }) {
+  const { title, text, url, button } = userData;
+  const [inputValue, setInputValue] = useState({ id: '', pw: '' });
+  const navigate = useNavigate();
+  const [validId, setValidId] = useState(false);
+  const [validPw, setValidPw] = useState(false);
+  const [termsAgree, setTermsAgree] = useState(false);
```

Comment:
```suggestion
  const [inputValue, setInputValue] = useState({ id: '', pw: '' });
  const [termsAgree, setTermsAgree] = useState(false);
  const navigate = useNavigate();
```

id와 pw의 valid를 판단하는 것은 변수로 관리할 수 있기에 state로 관리하지 않아도 됩니다!